packages/ui/src/AddressView/index.tsx

```diff
@@ -61,11 +61,13 @@ export const AddressViewComponent = ({
   const lqtLabel = isLqt ? 'My Tournament Address for' : '';
   const accountLabel = `${randomizedLabel}${lqtLabel} ${indexLabel}`;
 
-  const encodedAddress = external ? address.altBech32m : bech32mAddress(address);
+  // Treat addresses whose `inner` component is missing or incorrect length as external
+  const autoExternal = !address.inner || address.inner.length !== 80;
```

Comment:
@TalDerei how correct do you think this is? I mean, does length of Uint8Array of address.inner always 80?